.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
 		"**/.pnp.*": true
 	},
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
+		"source.fixAll.eslint": "explicit"
```

Comment:
Can revert this change. It likely will conflict. Instead merge upstream/master into your branch and this won't be something detected as a change.

Reply:
Handled by merging upstream master myself.